Cleaner_42.sh

```diff
@@ -44,49 +44,88 @@ echo -e "\033[33m\n -- Available Storage Before Cleaning : || $Storage || --\033
 
 echo -e "\033[31m\n -- Cleaning ...\n\033[0m "
 
-#42 Caches
-/bin/rm -rf "$HOME"/Library/*.42* &>/dev/null
-/bin/rm -rf "$HOME"/*.42* &>/dev/null
-/bin/rm -rf "$HOME"/.zcompdump* &>/dev/null
-/bin/rm -rf "$HOME"/.cocoapods.42_cache_bak* &>/dev/null
-
-#Trash
-/bin/rm -rf "$HOME"/.Trash/* &>/dev/null
-
-#General Caches files
-#giving access rights on Homebrew caches, so the script can delete them
-/bin/chmod -R 777 "$HOME"/Library/Caches/Homebrew &>/dev/null
-/bin/rm -rf "$HOME"/Library/Caches/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Caches/* &>/dev/null
-
-#Slack, VSCode, Discord and Chrome Caches
-/bin/rm -rf "$HOME"/Library/Application\ Support/Slack/Service\ Worker/CacheStorage/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Slack/Cache/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/discord/Cache/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/discord/Code\ Cache/js* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/discord/Crashpad/completed/*  &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Code/Cache/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Code/CachedData/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Code/Crashpad/completed/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Code/User/workspaceStorage/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Profile\ [0-9]/Service\ Worker/CacheStorage/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Default/Service\ Worker/CacheStorage/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Profile\ [0-9]/Application\ Cache/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Default/Application\ Cache/* &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Crashpad/completed/* &>/dev/null
-
-#.DS_Store files
-find "$HOME"/Desktop -name .DS_Store -depth -exec /bin/rm {} \; &>/dev/null
-
-#tmp downloaded files with browsers
-/bin/rm -rf "$HOME"/Library/Application\ Support/Chromium/Default/File\ System &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Chromium/Profile\ [0-9]/File\ System &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Default/File\ System &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Profile\ [0-9]/File\ System &>/dev/null
-
-#things related to pool (piscine)
-/bin/rm -rf "$HOME"/Desktop/Piscine\ Rules\ *.mp4
-/bin/rm -rf "$HOME"/Desktop/PLAY_ME.webloc
+
+should_log=0
+if [[ "$1" == "-p" || "$1" == "--print" ]]; then
```

Comment:
Can you note --print as well in the README?

Reply:
I'll do that first thing tomorrow.

Reply:
one thing I hesitated about is using the `getopt` built-in to extract the flag but I ended up using a simple check on the first script argument, should I reconsider implementing it? I think it'll be best for future features/options.

Reply:
I think a simple check would be enough to make it easier for other people to contribute.
Thank you